includes/datastream.inc

```diff
@@ -63,6 +63,11 @@ function islandora_view_datastream(AbstractDatastream $datastream, $download = F
     if ($duplicate_extension_position === FALSE) {
       $filename .= $extension;
     }
+
+    // Allow other modules to modify or replace the filename.
+    drupal_alter('datastream_filename', $filename, $datastream);
+
+
```

Comment:
From TravisCI: there is one empty line too many (multiple empty lines are not allowed).

@patdunlavey sincere apologies, I overlooked that one.

@DiegoPino @rosiel TravisCI is only silently complaining about it.

Reply:
Fixed. Thanks!